### ISSUE TITLE
Could you provide a 32-bit version?

### ISSUE THREAD

**Issue by tcbbd** [JustasMasiulis/ida_bitfields#3]
To use this plugin in x86 decompiler, I need a 32-bit version dll. I tried to compile myself but failed, seems it depends on the latest SDK.

**Comment by JustasMasiulis** [comment]
tried to do a quick compile, but encountered build issues as well. Will try to investigate and fix it.

**Comment by CookiePLMonster** [comment]
Bumping this issue to check what's the status of this - I wanted to try this plugin in hopes of helping me with bitfields in a 32-bit application, just to find out that this issue is the most likely blocker.

**Comment by JustasMasiulis** [comment]
Yeah, it's unlikely that I'll publish pre-built 32 bit version for some time. At the point the issue was made, I seemed to be missing some files from SDK for 32 bit builds and right now I don't even have this project set-up to be compiled.

**Comment by CookiePLMonster** [comment]
That's unfortunate, but understandable. Thanks for an update!

**Comment by JustasMasiulis** [comment]
Also, a small sidenote - 32-bit IDA is soon (tm) going to not be a thing [IDA 8.1 release notes](https://hex-rays.com/products/ida/news/8_1)

**Comment by CookiePLMonster** [comment]
> Also, a small sidenote - 32-bit IDA is soon (tm) going to not be a thing [IDA 8.1 release notes](https://hex-rays.com/products/ida/news/8_1)

> in the future 32-bit files will be analyzed and decompiled in IDA64.

Sounds like a non-issue for the time being, at least when it comes to analyzing 32-bit executables - hopefully support for those is kept forever, contrary to that these highlights imply.

**Comment by JustasMasiulis** [comment]
> > Also, a small sidenote - 32-bit IDA is soon (tm) going to not be a thing [IDA 8.1 release notes](https://hex-rays.com/products/ida/news/8_1)
> 
> > in the future 32-bit files will be analyzed and decompiled in IDA64.
> 
> Sounds like a non-issue for the time being, at least when it comes to analyzing 32-bit executables - hopefully support for those is kept forever, contrary to that these highlights imply.

Yup, my point was that maybe soon enough you'll be able to just use the provided 64-bit plugin to analyze 32-bit executables 👍

**Comment by CookiePLMonster** [comment]
> in the future 32-bit files will be analyzed and decompiled in IDA64.

Went back to check this and indeed, this issue is no longer relevant and most likely can be closed.
